draft-ietf-modpod-group-processes.md

```diff
@@ -229,6 +229,9 @@ their own judgment and community suggestions. (Note that some participation
 channels, such as the examples given in the previous sentence, have no
 chairs or other community leadership, so the moderators MUST handle those.)
 
+Moderators are expected to be a resource that group management can use
+to address problematic contributions.
```

Comment:
This needs to move down into "IETF Working Groups"?

Reply:
I pulled it forward because it seemed like part of the moderators' scope.  This may be one that cuts either way.  Whichever you prefer.

Reply:
It's just that "group management" hasn't been talked about before.

Reply:
Yeah... this sort of ties to the point that Jay and Brian are making about handling non-WG fora.  We need a bit more text, but I think that's a bit more substantial than this.  I don't want to back into that here...  I'll update with a small amount of text.